_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-minimal
```

Comment:
Is this done on purpose?

Reply:
I don't know, unfortunately, how it was added. 😕